programs/stake/src/stake_state.rs

```diff
@@ -553,9 +559,13 @@ impl<'a> StakeAccount for KeyedAccount<'a> {
             }
             StakeState::Stake(meta, mut stake) => {
                 meta.authorized.check(signers, StakeAuthorize::Staker)?;
+                let stake_amount = self.lamports()?.saturating_sub(meta.rent_exempt_reserve); // can't stake the rent ;)
+                if stake_amount < MINIMUM_STAKE_DELEGATION {
+                    return Err(InstructionError::StakeDelegationTooSmall);
+                }
```

Comment:
Comment here too, please

Reply:
Comment added! Can let me know what you think?